docs/conf.py

```diff
@@ -38,6 +38,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.viewcode',
     'sphinxcontrib.httpdomain',
+    'sphinx_rtd_theme',
```

Comment:
If this is needed for users to make Sphinx work you need to add it to the installing docs.

Reply:
You are right, just that first I want to show the way to resolve this improvement, If it's accepted, then I believe that these changes have to be documented.